spinalcordtoolbox/registration/labeling.py

```diff
@@ -15,9 +15,9 @@
 import spinalcordtoolbox.labels as sct_labels
 
 
-def add_orthogonal_label(fname_label):
+def add_orthogonal_label(fname_label, new_label_value):
     """
-    Add one label of value=99 at the axial slice that contains the label with the lowest value, 10 pixels to the right.
+    Add one label at the axial slice that contains the label with the lowest value, 10 pixels to the right.
```

Comment:
I know there's a TODO comment to make this code do "10 pixels to the right", but right now it does "5mm to the left", so the docstring should match the behaviour:
```suggestion
    Add one label at the axial slice that contains the label with the lowest value, 5mm to the left.
```